README.md

```diff
@@ -19,15 +19,24 @@ not be used in production.
 
 Ensure that Docker is installed and running on your machine before proceeding.
 
-#### Using Docker Compose File
-1. Create a `logging.ini` file from the example in the root of the project directory:
+1. Create a `.env` file alongside the `.env.example` file. Use the example file as a reference and modify the values accordingly.
+    ```bash
+    cp .env.example .env
+    ```
+
+2. Create a `logging.ini` file alongside the `logging.example.ini` file. Use the example file as a reference and modify it accordingly:
     ```bash
-   cp logging.example.ini logging.ini
+    cp logging.example.ini logging.ini
     ```
 
-2. Build and start the Docker containers:
+3. (**Required only if JWT Auth is enabled**) Inside the `keys` directory at the root of the project directory, create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.
```

Comment:
Could probably mention as well to ensure there is a `.keep` file, otherwise don't people risk just losing their public key as it is in the `.gitignore` file

Reply:
I don't think this is an issue, the .keep just ensures the folder is committed if git thinks its empty, which it will do since the jwt is in the git ignore. We don't want anyone committing their jwt public key accidentally.